meta-balena-common/recipes-core/systemd/disk-watchdog.bb

```diff
@@ -0,0 +1,50 @@
+SUMMARY = "Disk watchdog service for monitoring disk health"
+DESCRIPTION = "A watchdog service that monitors disk health"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${BALENA_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = "git://github.com/balena-os/disk-watchdogd.git;branch=add-flowzone;protocol=https"
```

Comment:
Is this supposed to be used from the `add-flowzone` branch or shall the code rather be moved to the `master` branch?